tox.ini

```diff
@@ -60,3 +47,31 @@ deps =
 commands =
     {[testenv:build]commands}
     twine upload --skip-existing dist/*
+
+[flake8]
+extend-ignore = 
+    E203, # whitespace before ':' 
+    E501, # line too long
+    C901, # is too complex
+    E402, # module level import not at top of file (for docs)
+    E741, # ambiguous variable name
+
+exclude =
+    .git,
+    __pycache__,
+    docs/source/conf.py,
+    old,
+    build,
+    dist,
+    .tox
+max-line-length = 88
+max-complexity = 10
+count=true
+
+[isort]
+profile = black
+
+[black]
+line-length = 88
+target-version = ['py37', 'py38', 'py39', 'py310']
+experimental_string_processing = true
```

Comment:
@rodrigoalmeida94 what do you think about moving those configurations (flake, isort, black) into a `setup.cfg` file. While having it in `tox.ini` is technically ok, I think `setup.cfg` is more supported.

Reply:
Good idea @vincentsarago !! Added.